### ISSUE TITLE
Widget timezone reporting

### ISSUE THREAD

**Issue by thejoshsmith** [bymayo/craft-commerce-widgets#32]
Hi there, would it be possible to take into account Craft's timezone setting when fetching data for the various widgets?

I recently took the plunge and built a custom widget that reported on different order types (specific to a particular CC site). I found that all dates are stored in UTC time, so to get the correct output for my timezone (Pacific/Auckland), I needed to do something similar to a fix implemented in CC earlier in the year: https://github.com/craftcms/commerce/commit/b36ad11ea21d1914edefa33003453301ea30f4c3.

Just for reference, this is what I'm currently seeing with the goal widget vs. Craft Commerce:

![Screen Shot 2019-10-15 at 9 05 34 AM](https://user-images.githubusercontent.com/5515179/66780197-b5404b80-ef2c-11e9-8f4c-611297d51197.png)

![Screen Shot 2019-10-15 at 9 05 26 AM](https://user-images.githubusercontent.com/5515179/66780201-ba9d9600-ef2c-11e9-89ec-202b902ad8a6.png)

... and this is from my custom widget:

![Screen Shot 2019-10-15 at 9 05 41 AM](https://user-images.githubusercontent.com/5515179/66780207-be311d00-ef2c-11e9-96aa-095d6b1801ba.png)

If I get time I might have a crack and raise a PR, but can't promise anything!

**Comment by bymayo** [comment]
Definitely! I do have timezones on the roadmap for this, but it wasn't a priority, but I can bump it up.

If you fancy doing a PR though, go ahead, as it will be a few weeks until I can look at adding this in. If you do a PR I'll chuck you a free license 💯

**Comment by bymayo** [comment]
@moacode I know this is an old issue, but this has now been fixed in 5.1.0.

Please update this plugin, and make sure your project is running Craft CMS 5 and Craft Commerce 5 to get this fix
